src/components/chat/SettingsPanelOld.tsx

```
@@ -0,0 +1,555 @@
+                            initial={{ opacity: 0, height: 0 }}
+                            animate={{ opacity: 1, height: 'auto' }}
+                            exit={{ opacity: 0, height: 0 }}
+                            className="space-y-3 pt-3 border-t border-current/10"
+                          >
+                            <div className="grid grid-cols-2 gap-3">
+                              <div>
+                                <label className={`text-xs font-medium mb-1 block ${
+                                  isDarkMode ? 'text-gray-300' : 'text-gray-700'
+                                }`}>
+                                  Connection Type
+                                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)